packages/static-object-s3/src/index.ts

```diff
@@ -0,0 +1,28 @@
+import { type PresignedPost, createPresignedPost } from "@aws-sdk/s3-presigned-post"
+import { S3Client } from "@aws-sdk/client-s3"
+
+export interface S3Repository {
+  getPresignedPostData(bucket: string, filename: string, mimeType: string, maxSizeMB: number): Promise<PresignedPost>
+}
+
+export class S3RepositoryImpl implements S3Repository {
```

Comment:
I think this can just be moved to core

Reply:
I agree. It didn't really fit in `core/modules`, as what's there is just db resources, so I put it in `/lib`